config/kind/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+bases:
```

Comment:
based on the other PR, let not use bases and move the path to ../base under the resources.

Reply:
done